diff -Nur jikes-1.13.orig/src/stream.cpp jikes-1.13/src/stream.cpp
--- jikes-1.13.orig/src/stream.cpp	Sat Feb 24 22:50:16 2001
+++ jikes-1.13/src/stream.cpp	Mon Mar  5 11:42:31 2001
@@ -326,7 +326,7 @@
         return true;
     }
 #elif defined(HAVE_ICONV_H)
-    _converter = iconv_open("utf-16", encoding);
+    _converter = iconv_open("UTF-16BE", encoding);
     if (_converter == (iconv_t)-1)
     {
         return false;
