--- jikes-1.12/src/stream.cpp.chmou	Tue Jul 25 13:32:33 2000
+++ jikes-1.12/src/stream.cpp	Fri Nov 17 18:13:00 2000
@@ -890,7 +890,7 @@
                     size_t   chl  = 2;
                     size_t   srcl = 1;
                     size_t n = iconv(control.option.converter,
-                                     &source_ptr, &srcl,
+                                     (char **)&source_ptr, &srcl,
                                      (char **)&chp, &chl
                     );
 
