--- jikes-1.12/src/option.cpp.orig	Tue Nov 28 14:35:22 2000
+++ jikes-1.12/src/option.cpp	Tue Nov 28 14:36:07 2000
@@ -251,7 +251,7 @@
 #elif defined(HAVE_ICONV_H)
                 encoding = new char[strlen(arguments.argv[++i]) + 1];
                 strcpy(encoding, arguments.argv[i]);
-                converter=iconv_open("utf-16", encoding);
+                converter=iconv_open("UTF-16BE", encoding);
                 if(converter==(iconv_t)-1)
                 {
                     converter = NULL;
