diff -ru jikes-1.12.orig/src/tuple.h jikes-1.12/src/tuple.h
--- jikes-1.12.orig/src/tuple.h	Tue Jul 25 13:32:34 2000
+++ jikes-1.12/src/tuple.h	Tue Aug  8 17:34:08 2000
@@ -92,7 +92,16 @@
         // instead of having to perform a subtraction for each reference.
         // See operator[] below.
         //
-        base[k] = new T[Blksize()];
+
+	// Workaround for a compiler bug (gcc 2.95.3 choke on the original code)
+	// MDC, Mandrakesoft
+
+	int blkSize = Blksize();
+	base[k] = new T[blkSize];
+
+	// Original code
+	// base[k] = new T [Blksize()];
+
         base[k] -= size;
 
         //
Only in jikes-1.12/src: tuple.h~
