--- jikes-1.15/src/double.h~	Sat Sep 29 03:12:19 2001
+++ jikes-1.15/src/double.h	Fri Dec 14 00:28:10 2001
@@ -112,7 +112,7 @@
         POS_ZERO   = 0x00000000, // 0.0
         NEG_INF    = 0xFF800000, // -Inf
         POS_INF    = 0x7F800000, // +Inf
-        NAN        = 0x7FC00000, // canonical NaN
+        NAN_       = 0x7FC00000, // canonical NaN
         BYTE_MASK  = 0x000000FF, // mask off least significant byte
         MAX_INT    = 0x7FFFFFFF, // maximum integer
         MIN_INT    = 0x80000000, // minimum integer
