diff -uNr jikes-1.12.orig/src/stream.cpp jikes-1.12/src/stream.cpp
--- jikes-1.12.orig/src/stream.cpp	Tue Jul 25 07:32:33 2000
+++ jikes-1.12/src/stream.cpp	Thu Aug  3 11:42:03 2000
@@ -890,7 +890,7 @@
                     size_t   chl  = 2;
                     size_t   srcl = 1;
                     size_t n = iconv(control.option.converter,
-                                     &source_ptr, &srcl,
+                                     (char **)&source_ptr, &srcl,
                                      (char **)&chp, &chl
                     );
 
